### PR TITLE
docs: minor fixes

### PR DESCRIPTION
**Pull request by mhrabovcin**

### CODE REVIEW

File: docs/pages/developer-guide.md [hunk 1]
```diff
@@ -1424,6 +1424,7 @@ pods:
 
 The path is relative to the sandbox path if not preceded by a leading "/". The sandbox path is always available in the environment variable MESOS_SANDBOX.  The different between ROOT and MOUNT volumes is [documented here](http://mesos.apache.org/documentation/latest/multiple-disk/). The PATH type is not currently supported.
 
+<a name="proxy-fallback"></a>
 ### Proxy Fallback
 
 Applications may not work properly behind adminrouter. In that case, one may use [Repoxy](https://gist.github.com/nlsun/877411115f7e3b885b5e9daa8821722f).
```
Comment (sascala) [inline docs/pages/developer-guide.md]: Is the name `Repoxy`, as named in Nick's gist? Or is it `Reproxy`, as named in `yams-reference.md`, below?

**Comment by ArmandGrillet** [comment]
There is a very small typo that could also be fixed in this PR, line 41 of `docs/pages/developer-guide.md`:

> every scheduler must keep a record of **the the** state of its tasks.

Would be awesome to fix the typo here, otherwise I'll create a PR if necessary.

**Comment by gabrielhartmann** [comment]
@mhrabovcin and @ArmandGrillet:  Sorry for the delay on attention for this PR. Is it possible to resolve the conflicts / comments so we can merge this?

**Comment by mhrabovcin** [comment]
@gabrielhartmann I've rebased changes on top of `master` branch.

**Pull request merged**